Solutions/FlareSystemsFirework/Data Connectors/Connector_REST_API_FlareSystemsFirework.json

```diff
@@ -73,7 +73,29 @@
                     "description": "As an organization administrator, authenticate on [Firework](https://firework.flared.io/) and access the [team page](https://firework.flared.io/#/team) to create a new alert channel."
                 },
                 {
-                    "description": "Click on 'Create a new alert channel' and select 'Azure Sentinel'. Enter your Shared Key And WorkspaceID. Save the Alert Channel."
+                    "description": "Click on 'Create a new alert channel' and select 'Azure Sentinel'. Enter your Shared Key And WorkspaceID. Save the Alert Channel. \n For more help and details, see our [Azure configuration documentation](https://docs.microsoft.com/en-us/azure/sentinel/connect-data-sources).",
```

Comment:
Validation is failing because the Azure Configuration documentation link contains locale **en-us**. Please remove it.
suggestion for url that can be used is : **https://docs.microsoft.com/en-us/azure/sentinel/connect-data-sources**